src/etc/inc/auth.inc

```diff
@@ -488,7 +488,7 @@ function local_user_del($user)
 
 function local_user_set_password(&$user, $password)
 {
-    $user['password'] = crypt($password, '$6$');
+    $user['password'] = generate_password_hash($password, 10);
```

Comment:
Fair point on 10 vs. 12, ok. What is 10 in numbers on the same box to compare? :)

The prototype for password_hash() is pretty good, generate_password_hash() is too oriented towards the newly implemented default. In the future, there may be more calls from plugins, we can't expect to fix all of them because we e.g. want to bump the default from 10 to something else in the future. We may also have to dash out MD5 for non-local credentials, ideally routing all of this through the same function while offering sane defaults underneath is desirable.